WordPress/Classes/Extensions/UITableView+Helpers.swift

```diff
@@ -31,4 +39,17 @@ extension UITableView
             self.deselectSelectedRowWithAnimation(animated)
         }
     }
+
+    /// Disables Editing after a specified delay.
+    ///
+    /// -   Parameter delay: milliseconds to elapse before edition will be disabled.
+    ///
+    public func disableEditionAfterDelay(delay: NSTimeInterval = 0.2) {
```

Comment:
`delay: NSTimeInterval = 0.2`

Why are we setting the parameter value here? And please set `0.2` to a constant.